openqaoa/problems/converters.py

```diff
@@ -327,7 +330,7 @@ def qubo_to_ising(n_variables, qubo_terms, qubo_weights):
 
         return QUBO(n_variables, ising_terms, ising_weights)
 
-    def get_models(self, multipliers: float = None):
+    def get_models(self, multipliers: [float, list] = None):
```

Comment:
Same issue as the init function for this input parameter

Reply:
Thanks @shahidee44, I updated it!